test/unit-tests/components/views/dialogs/__snapshots__/DevtoolsDialog-test.tsx.snap

```diff
@@ -105,117 +105,127 @@ exports[`DevtoolsDialog renders the devtools dialog 1`] = `
-            id="mx_SettingsFlag_QgU2PomxwKpa"
-            role="switch"
-            tabindex="0"
+            class="_inline-field-body_19upo_38"
           >
-            <div
-              class="mx_ToggleSwitch_ball"
-            />
+            <label
+              class="_label_19upo_59"
+              for="mx_SettingsFlag_QgU2PomxwKpa"
+            >
+              Show hidden events in timeline
+            </label>
           </div>
         </div>
         <div
-          class="mx_SettingsFlag"
+          class="_inline-field_19upo_32"
         >
-          <label
-            class="mx_SettingsFlag_label"
-            for="mx_SettingsFlag_6hpi3YEetmBG"
-          >
-            <span
-              class="mx_SettingsFlag_labelText"
-            >
-              Enable widget screenshots on supported widgets
-            </span>
-          </label>
           <div
-            aria-checked="false"
-            aria-disabled="false"
-            aria-label="Enable widget screenshots on supported widgets"
-            class="mx_AccessibleButton mx_ToggleSwitch mx_ToggleSwitch_enabled"
-            id="mx_SettingsFlag_6hpi3YEetmBG"
-            role="switch"
-            tabindex="0"
+            class="_inline-field-control_19upo_44"
           >
             <div
-              class="mx_ToggleSwitch_ball"
-            />
+              class="_container_19o42_10"
+            >
+              <input
+                class="_input_19o42_24"
+                id="mx_SettingsFlag_6hpi3YEetmBG"
+                role="switch"
+                type="checkbox"
+              />
+              <div
+                class="_ui_19o42_34"
+              />
+            </div>
           </div>
-        </div>
-        <form
-          class="_root_19upo_16"
-        >
           <div
-            class="_field_19upo_26"
+            class="_inline-field-body_19upo_38"
           >
             <label
               class="_label_19upo_59"
-              for="radix-«r4»"
+              for="mx_SettingsFlag_6hpi3YEetmBG"
             >
-              Element Call URL
+              Enable widget screenshots on supported widgets
             </label>
-            <div
-              class="_controls_17lij_8"
-            >
-              <input
-                class="_control_sqdq4_10"
-                id="radix-«r4»"
-                name="input"
-                title=""
-                value=""
-              />
-            </div>
           </div>
-        </form>
-      </div>
+        </div>
+        <div
+          class="_field_19upo_26"
+        >
+          <label
+            class="_label_19upo_59"
+            for="radix-«ra»"
+          >
+            Element Call URL
+          </label>
+          <div
+            class="_controls_17lij_8"
+          >
+            <input
+              class="_control_sqdq4_10"
+              id="radix-«ra»"
+              name="input"
+              title=""
```

Comment:
this seems odd, surely if there's no title the attribute should be omitted rather than an empty string?